### PR TITLE
ESS-1256: Styling for Non Privileged Auto-priv Styles, and refactorings

### PR DESCRIPTION
**Pull request by leonardotemasys**

### CODE REVIEW

File: demo/privileged/auto-unpriv/index.html [hunk 1]
```diff
@@ -95,125 +95,157 @@
...
+              </div>
+            </div>
           </div>
         </div>
       </div>
+    </section>
+    <div class="">
+      <div class="container">
+        <p class="text-center">
+          &copy; Temasys Communications Pte Ltd 2016. All Rights Reserved.
```
Comment (ishannaithani) [inline demo/privileged/auto-unpriv/index.html:245]: Can be changed to Temasys Communications Pte Ltd **2018** wherever you encounter

**Comment by ishannaithani** [comment]
Changes look good!

File: demo/privileged/auto-unpriv/index.html (hunk 1 above)
Comment (leonardotemasys) [reply demo/privileged/auto-unpriv/index.html:245]: Sure

**Comment by leonardotemasys** [comment]
Should I merge @ishannaithani ?

**Pull request merged**